tests/test_plugins.py

```diff
@@ -49,3 +64,86 @@ def test_select_attr(project):
     )
 
     assert sorted(project.get_instances("__config__::Out")[0].fields) == ["A", "B", "C"]
+
+
+def run_test(project, thetype, value, is_ok):
+    def make():
+        project.compile(
+            f"""
+import std
+entity Holder:
+    {thetype} value
+end
+implement Holder using std::none
+
+Holder(value="{value}")
+"""
+        )
+
+    if not is_ok:
+        with pytest.raises(RuntimeException):
+            make()
+    else:
+        make()
+
+
+def test_hostname(project):
+    hostname = "test"
+    fqdn = f"{hostname}.something.com"
+    assert project.get_plugin_function("hostname")(fqdn) == hostname
+
+
+def test_cidr_to_prefixlen(project):
+    cidr = "192.168.1.100/24"
+    prefixlen = "24"
+    assert project.get_plugin_function("cidr_to_prefixlen")(cidr) == prefixlen
```

Comment:
Let's also simplify the test cases. It makes no sense to have this variables it only reduces readability.

```suggestion
    assert project.get_plugin_function("cidr_to_prefixlen")("192.168.1.100/24") == "24"
    
```